### DISCUSSION TITLE
Gray weekends

### DISCUSSION THREAD

**Discussion by vasco9999** [kudrykv/latex-yearly-planner#66] [Ideas]
would it be interesting to have weekend gray in the different calendar views

**Comment by kudrykv** [comment]
Can you provide an example? If you're talking about the daily view, that's WIP in the refactor branch (don't mind the red highlights, that's for my debugging)
![CleanShot 2022-12-06 at 20 09 07](https://user-images.githubusercontent.com/12697803/205988766-980344cc-6b5d-4160-82d6-7a556db8ea53.png)

**Comment by vasco9999** [comment]
I was thinking about the quarterly planner that I proposed. For the month view something like this? It is just a small idea, can be an improvement or for some a disadvantage.

![image](https://user-images.githubusercontent.com/119439720/206173710-1bf11171-609f-4bb2-8192-5877bce6a46b.png)

**Reply by kudrykv** [reply]
Do you mean something like having a little calendar for the monthly view, and having todos, notes, etc., for the rest of the page?

**Comment by vasco9999** [comment]
yes, that could be the case
